package.json

```diff
@@ -245,6 +245,7 @@
     "@types/react-collapse": "^5.0.1",
     "@types/react-dom": "^18.2.4",
     "@types/react-is": "^18.3.0",
+    "@types/react-native-web": "^0.0.0",
```

Comment:
@blazejkustra Is this really correct?

Reply:
Yes 😅 DefinietlyTyped contributor never responded [here](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/70147#discussion_r1696047739)